src/pkg/caendr/caendr/models/run/__init__.py

```diff
@@ -2,6 +2,3 @@
 from .base   import Runner
 from .gcp    import GCPRunner, GCPCloudRunRunner, GCPLifesciencesRunner
 from .local  import LocalRunner
-
-# Job-specific subclasses
```

Comment:
These are deprecated in favor of using the generic `Runner` class, and defining any parameters in the job-specific `JobPipeline` class.

The `JobPipeline` class primarily mediates between a given `Report` and a given `Runner`, e.g. reading props from the `Report` to construct the run environment.